salt/metalk8s/roles/etcd/prepared.sls

```diff
@@ -0,0 +1,7 @@
+# add member to cluster before etcd starts
+Register the node into etcd cluster:
+  salt.runner:
+    - name: state.orchestrate
+    - pillar: {{ pillar | json  }}
+    - mods:
+      - metalk8s.orchestrate.register_etcd
```

Comment:
Maybe an `include` of this state is enough here.

Also it's weird to me to have a sls in `roles` directory that need to run on the salt-master (cc @NicolasT )